.pre-commit-config.yaml

```diff
@@ -11,31 +11,39 @@ repos:
       - id: trailing-whitespace
         args: [--markdown-linebreak-ext=md]
 
+  - repo: local
+    hooks:
+      - id: no-symlinks
+        name: no symlinks
+        description: "Check for symlinks"
+        entry: ".*"
```

Comment:
This entry should be a description of what this hook does: its printed when this hook fails.
https://pre-commit.com/#fail

Reply:
Additionally, in a separate pr it would be worth adding the require rst files check in the changelog directory like documented in the link above.

Reply:
Oops, good catch. Not sure how that happened
https://github.com/DisnakeDev/disnake/pull/700/commits/fde2c387903859f3897082410889dcfa091d6e14